app/views/projects/tree/_item.html.haml

```diff
@@ -0,0 +1,8 @@
+%tr{ class: "tree-item #{tree_hex_class(item)}" }
+  %td.tree-item-file-name
+    = tree_icon(type)
+    %span.str-truncated
+      = link_to item.name, send(path_helper, @project, tree_join(@id || @commit.id, item.name))
```

Comment:
Just to double check, in the `_blog_item.html.haml` is the `project_blob_path` but in the `_tree_item.html.haml` the `project_tree_path` is used. And in this file none if them, can you explain a little what happends here?